### ISSUE TITLE
[bug] certbot-renew is blocked by bad UA list and bad referer list

### ISSUE THREAD

**Issue by thelittlefireman** [bunkerity/bunkerized-nginx#100]
hi,

certbot autorenew fails everytimes due to bad UA list and bad referer list set to `yes`. Could id be possible to disable theses list before renew ?

```
2021/03/13 17:45:07 [warn] 487#487: *760 [lua] access_by_lua(main-lua.conf:166):80: [BLOCK] User-Agent Mozilla/5.0 (compatible; Let's Encrypt validation server; +https://www.letsencrypt.org) is blacklisted, client: XXXXXXXXX, server: nextcloud.mydomain, request: "GET /.well-known/acme-challenge/a6R7xNKyHHc5wLP8mLKjIWoO_O2gPfpHhmSOs-1Ywmk HTTP/1.1", host: "nextcloud.mydomain", referrer: "http://nextcloud.mydomain/.well-known/acme-challenge/a6R7xNKyHHc5wLP8mLKjIWoO_O2gPfpHhmSOs-1Ywmk"
2021/03/13 17:45:25 [warn] 496#496: *798 [lua] access_by_lua(main-lua.conf:166):86: [BLOCK] Referrer http://syno.mydomain/.well-known/acme-challenge/iuwgjj22HlM4MFw-Hj4Vn4wpfwAqFTxOQiZfkQEOIFg is blacklisted, client: XXXXXXXX, server: syno.mydomain, request: "GET /.well-known/acme-challenge/iuwgjj22HlM4MFw-Hj4Vn4wpfwAqFTxOQiZfkQEOIFg HTTP/1.1", host: "syno.mydomain", referrer: "http://syno.mydomain/.well-known/acme-challenge/iuwgjj22HlM4MFw-Hj4Vn4wpfwAqFTxOQiZfkQEOIFg"
```

To reproduce enable `AUTO_LETS_ENCRYPT=yes` `BLOCK_REFERER=yes` and `BLOCK_USER_AGENT=yes`
connect as root on the container, and execute `/opt/scripts/certbot-renew.sh` ==> renew challenges fail
if you change to `BLOCK_REFERER=no` and `BLOCK_USER_AGENT=no`==> renew challenges succed

i don't understand why referer block all mydomain, they are not in the list [https://raw.githubusercontent.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/master/_generator_lists/bad-referrers.list](https://raw.githubusercontent.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/master/_generator_lists/bad-referrers.list)

**Comment by thelittlefireman** [comment]
Good idea ! It will be a better fix than my PR ;)

**Comment by bunkerity** [comment]
Hey @thelittlefireman,

Should be fixed in the last 1.2.3 version. Thanks.
